### PR TITLE
Cultists of Nar-sie have their own language

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Cultists of Nar-sie have their own language. The language's key is ",n".
/:cl:

Scrambled phrases are now per-language, instead of across all languages, and more scrambled phrases will be saved.

### CODE REVIEW

**Comment by Armhulen** [comment]
🎉

**Comment by coiax** [comment]
@WJohn you can make a new language badge for us if you want

File: code/modules/language/language.dm [hunk 1]
```diff
@@ -16,10 +16,10 @@
 	// If key is null, then the language isn't real or learnable.
 	var/flags                         // Various language flags.
 	var/list/syllables                // Used when scrambling text for a non-speaker.
-	var/list/sentence_chance = 5      // Likelihood of making a new sentence after each syllable.
-	var/list/space_chance = 55        // Likelihood of getting a space in the random scramble string
```
Comment (coiax) [inline code/modules/language/language.dm]: whoops

**Comment by InsaneHyena** [comment]
So, will the cultists understand their own spellcasting now?

**Comment by coiax** [comment]
No, the spell casting is said in common.

**Comment by ExcessiveUseOfCobblestone** [comment]
would be cool if we they had english translations in Narsian but looked like they do currently in GalCom

**Comment by coiax** [comment]
Most of the invocations are just memes or references and have no original english text.

**Comment by ChangelingRain** [comment]
Yeah, I would if I could but there IS no original text.

**Comment by Robustin** [comment]
@ChangelingRain comes up with exciting new ways to conflict my Cult PR every day!

On a serious note, will this effect cult slurring? I threw together the cult slur in like 5 minutes and it went surprisingly well... for 5 minutes of work. It could really use a more convincing mingling of languages.

**Comment by ChangelingRain** [comment]
it doesn't affect cultslurring, no, and I'm not sure how to make it do so offhand either.

**Comment by coiax** [comment]
I like the current cult slurring effect, leave it be

**Comment by XDTM** [comment]
It would be fun tho if people affected by cultslurring forcefully spoke in narsian, might lead to situations where an escapee gets taken for a cultist, or viceversa a caught cultists can pretend to be affected by a talisman

**Comment by KorPhaeron** [comment]
Missing a "why" in the op

**Comment by ChangelingRain** [comment]
Parity with clockcult, which gets its own language with the same amount of tells.
Also I thought it'd be cool, and it is.

**Comment by Armhulen** [comment]
((((it is))))

File: code/modules/surgery/organs/tongue.dm [hunk 2]
```diff
@@ -15,6 +15,7 @@
 		/datum/language/common,
 		/datum/language/draconic,
 		/datum/language/monkey,
+		/datum/language/narsie,
```
Comment (PKPenguin321) [inline code/modules/surgery/organs/tongue.dm]: this whole list should really be made a define or something in the actual `language` folder but that's kind of unrelated to this PR

**Comment by KorPhaeron** [comment]
"its cool" and "an unrelated mode does it" arent really reasons but this will have an entirely negligible gameplay impact so whatever

**Comment by KorPhaeron** [comment]
Will this conflict with your cult fix I feel like the mode functioning should have priority

**Comment by ChangelingRain** [comment]
It won't conflict no

**Comment by KorPhaeron** [comment]
@coiax do the language changes look okay to you I am outsourcing my review

**Comment by coiax** [comment]
Looks fine to me.

**Pull request merged**